Mathlib/Algebra/Opposites.lean

```diff
@@ -205,6 +205,9 @@ instance unique [Unique α] : Unique αᵐᵒᵖ :=
 instance isEmpty [IsEmpty α] : IsEmpty αᵐᵒᵖ :=
   Function.isEmpty unop
 
+@[to_additive]
+instance instDecidableEq [DecidableEq α] : DecidableEq αᵐᵒᵖ := fun _ _ ↦ decidable_of_iff _ unop_inj
```

Comment:
What do you think about this?
```suggestion
instance instDecidableEq [DecidableEq α] : DecidableEq αᵐᵒᵖ := unop_injective.decidableEq
```

Reply:
Sure yeah! It should give the same algorithm in the end, so it's okay.